Code/GraphMol/MolStandardize/Metal.h

```diff
@@ -31,14 +31,28 @@ namespace MolStandardize {
     -
 */
 
+struct RDKIT_MOLSTANDARDIZE_EXPORT MetalDisconnectorOptions {
+  bool splitGrignards =
```

Comment:
you don't need to include the `default` part of the comments here: the default value is right there in the code